doc/nrf/libraries/others/app_event_manager.rst

```diff
@@ -141,10 +141,11 @@ To create a source file for the event type you defined in the header file:
 1. Include the header file for the new event type in your source file.
 #. Define the event type with the :c:macro:`APP_EVENT_TYPE_DEFINE` macro.
    Pass the name of the event type as declared in the header along with additional parameters.
-   For example, you can provide a function that fills a buffer with a string version of the event data (used for logging).
+   For example, you can provide a function that logs using event manager macro a string version of the event data (used for logging).
```

Comment:
`For example, you can provide a function that logs a string version of the event data using :c:macro:`APP_EVENT_MANAGER_LOG` macro`